src/nuget/Microsoft.DotNet.BuildTools.Run.nuspec

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>Microsoft.DotNet.BuildTools.Run</id>
+    <version>1.0.0-prerelease</version>
+    <title>Microsoft DotNet Run Tool</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=518631</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <summary>.NET Core Framework Run Tool</summary>
+    <description>This package provides build-time support for the .NET Core Framework libraries.  You should not need to reference this package from your project.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags></tags>
+    <dependencies>
+    <!--
+      Do not add any dependencies to this package as we want to install it versionless
+      (i.e. -ExcludeVersion) and we don't want the dependencies to be installed that way
+    -->
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="run\run.exe" target="tools\netcoreapp1.0" />
```

Comment:
We don't have a great story here. What do you mean by restoring a runtime and host? Do you mean the full shared FX? In either case you still have to worry about compatibility. The most resilient way would be to include everything in the package but then we would end up with a lot of duplication among our tools. I do think a shared framework is the best way we just have to figure out how we want to run on that without depending on a runtimeconfig file.